### DISCUSSION TITLE
Why does GitHub have the ability to add anyone as a co-author?

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [community/community#28740] [New to GitHub] [answered]
Why does GitHub have the ability to add anyone as a co-author? 
I tried to add Linus Torvalds as co-author to a commit and now it shows him as a contributor to my repo. But that shouldn't be the case, right?  

What if someone tries to add me as a coauthor for some repo?
Shouldn't an email confirmation be there asking whether I really want to be added as a co-author for a particular commit?

**Comment by mldyh** [comment, marked as answer]
The idea is to give a credit to all developers who help us deliver the code. It's not rare nowadays to do peer programming or at least took a help from other to deliver a task. For me, it is just small appreciation of their contribution. 

If you talked about someone put your acc as co-author of bad commit, I cant agree more that it should not be like this and so the same with git blame. You can find tons of threads discussing about git blame. But Linus name things after himself anyway 🤣
